### DISCUSSION TITLE
php artisan make:filament-resource  model as flag

### DISCUSSION THREAD

**Discussion by pcfjojo** [filamentphp/filament#13470] [Ideas]
Is it possible to create filament recource by passing the model or extended models

Example:
php artisan make:filament-resource  Investor --model=App\Models\User

php artisan make:filament-resource  Investor --model=App\Models\Investor

php artisan make:filament-resource  Investor --model=App\Models\Customer 

Let us say Investor and Customer models  extend user.

**Comment by leandrocfe** [comment]
No, I guess. The Model name is the first param, but you can edit it in the Resource class
